Source/GSRunLoopWatcher.m

```diff
@@ -41,7 +42,7 @@ - (id) initWithType: (RunLoopEventType)aType
 	       data: (void*)item
 {
   _invalidated = NO;
-  receiver = anObj;
+  receiver = RETAIN(anObj);
```

Comment:
We should scrap all this change:  the receiver needs to be un-retained in order to prevent retain cycles which would cause memory leaks.  I have corrected the comments in trunk to say this.